velox/exec/LocalPlanner.cpp

```diff
@@ -349,7 +337,7 @@ std::shared_ptr<Driver> DriverFactory::createDriver(
           ctx.get(),
           assignUniqueIdNode,
           assignUniqueIdNode->taskUniqueId(),
-          getAssignUniqueIdCounter(assignUniqueIdNode->id())));
```

Comment:
The code semantics are changing here. Previously, all nodes with the same `planNodeId` would return the same counter. But now, each instance of `assignUniqueIdNode()` will return a different counter irrespective of `planNodeId`. I assume this is logically fine since all we need is a unique id.

Reply:
@majetideepak Wonder in which case multiple instances of the same planNodeId will be created? The expectation is that for the assignUniqueIdNode with the same`planNodeId`, the counter will be shared so no id conflict

Reply:
@frankobe Makes sense that multiple instances of the same `planNodeId` will probably never happen.